README.md

```diff
@@ -396,6 +396,259 @@ public CloudTasksClient getService() throws IOException {
+A service *may* choose to only enable retries for a subset of RPCs. It is possible that for a single service, 
+each RPC is configured differently.
+
+### Google Cloud Client Library Retry Concepts
+Enabling retries allow an RPC multiple attempts to try and achieve a successful call. A successful call
+is a response from a server that returns an `OK` Status Code (from gRPC) or a `2xx` Status Code (from HttpJson).
+
+An RPC will be retried when _both_ of the following scenarios occur:
+- Non-successful status code is received by the library and the status code is marked as retryable*
+- An RPC invocation exceeds the individual RPC bounds, but still falls within total RPC bounds**
+
+Note: If only one (or neither) of the scenarios above are true, then the RPC will not be retried.
+i.e. If the total timeout has not been exceeded, but the latest attempt receives a non-retryable status code.
+
+*The client library will mark a status code as retryable internally. It is marked as retryable if the response's
+status code matches with any of an RPC's configure retryable status codes.
+
+**When configuring the RPC bounds, you may configure the bounds for each attempt as well as the
+total RPC's bounds. The retry algorithm will ensure that an individual attempt's bounds falls within
+the total RPC's bounds.
+
+#### Configurable Retry Params
+- Retry Status Code: Set of failure status codes to retry on
+- Retry Time/ Attempt Bounds: Configurable retry settings (via RetrySettings class) to define the retry bounds
+
+#### RetrySettings Configurable Params
+- Max Attempts: The maximum number of attempts to perform. If this value is greater than 0, and the number of attempts reaches this limit, the logic will give up retrying even if the total retry time is still lower than TotalTimeout.
+- Total Timeout: The overall total limit before the remote call gives up completely. The higher the total timeout, the more retries can be attempted.
+- Initial Retry Delay: The delay before the first retry.
+- Retry Delay Multiplier: The change in retry delay. This value is multiplied by the previous call’s retry delay to calculate the retry delay for the next call.
+- Max Retry Delay: The limit on the value of the retry delay, so that the RetryDelayMultiplier can't increase the retry delay higher than this amount.
+- Initial RPC Timeout: The timeout for the initial RPC.
+- RPC Timeout Multiplier: The change in RPC timeout. This value is multiplied by the previous call’s timeout to calculate the timeout for the next call.
+- Max RPC Timeout: The limit of the RPC timeout, so that the RpcTimeoutMultiplier can't increase the RPC timeout higher than this amount
+
+Note: It is recommended to only set _either_ the Max Attempt or the Total Timeout.
+
+See the official Google Cloud Docs: https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.retrying.RetrySettings
+
+#### Jitter
```

Comment:
I don't think we should mention `Jitter` here because it is already deprecated and we should always set it to true.

Reply:
I think disable Jitter internally quite a bit for tests to manage flakiness. I don't know if we can fully remove it in the future (perhaps we can just mark it with `@VisibleForTests`).

I mentioned this section since I think it has a bit of behavior impact on the number of retries. If a customer is either counting (or just otel), they may be expecting a certain number of retry attempts.